dockers/docker-lldp/lldpmgrd

```diff
@@ -219,6 +242,18 @@ def main():
 
     lldpmgr.run()
 
+def resume_lldpd(self):
+    cmd = "lldpcli resume"
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr) = proc.communicate()
+    if proc.returncode != 0:
+        self.log_error("Failed to resume lldpd with command: '{}': {}".format(cmd, stderr))
+        sys.exit(1)
+
+def check_timeout(self, start_time):
+    if time.time() - start_time > PORT_INIT_TIMEOUT:
+        self.log_error("Port init timeout reached ({} seconds), killing lldpmgrd...".format(PORT_INIT_TIMEOUT))
+        sys.exit(1)
```

Comment:
Please Just resume the process to make the behavior as it was before

Reply:
Fixed, please check